package/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# Package container with fpm (Linux: alpine) for virtualenv-tools
+# Supported workflow:
+# - Use fpm for python to .deb packaging
+
+# Why alpine?
+# - ruby: => @start ~750MB @end NoData
+# - ruby slim: +> @start ~280MB @end NoData
+# - debian wheezy => @srat ~85MB @end ~280MB
+# - ruby alpine: => @start ~5MB @end ~160MB
+# Why NOT alpine, maybe?
+# - new Linux flavour, new things to learn
+
+FROM ruby:2.3.1-alpine
+
+# Throw errors if Gemfile has been modified since Gemfile.lock.
+RUN bundle config --global frozen 1
+
+COPY Gemfile /
+COPY Gemfile.lock /
+
+### FPM installation ###
+# Package required for:
+# - binutils: provides ar, required by fpm (dir->deb)
```

Comment:
Actually, you do `python -> deb` instead of `dir -> deb`. Maybe, I would update this comment to be more generally: `ar` is required by `fpm` to build `.deb` packages.